agent/rpm/pbench-agent.spec.j2

```diff
@@ -34,6 +41,7 @@ Requires:  python3-cffi, python3-requests
 Requires:  python3-docutils, python3-psutil
 %endif
 
+
```

Comment:
Did you intend to add an extra blank line here?

Reply:
I wanted a larger break between RHEL and Fedora stuff. I find it helpful, but YMMV.